recipe/meta.yaml

```diff
@@ -28,8 +28,8 @@ app:
 
 requirements:
   host:
-    - jupyter-packaging >=0.9,<2
-    - packaging
+    - jupyter-packaging 0.12.0
+    - packaging 22.0
```

Comment:
I think packaging without version is okay. I will add it to the exception list in the linter.